app-copy.js

```diff
@@ -0,0 +1,79 @@
+const express = require('express');
+const bodyParser = require('body-parser');
+const path = require('path');
+const expressValidator = require('express-validator');
+const mongojs = require('mongojs');
+
+const db = mongojs('quotesapp', ['users']);
```

Comment:
I wonder if you could have used `['quotes']` instead of `['users']` here. If you have an object named `users` it usually communicates an intent that the data in the object is *only* about users. However if you have an object named `quotes` then it communicates that the data will be about quotes, and it’s reasonable to infer that a `quote` object would have a `user` attribute.